src/systemcmds/param/param.cpp

```diff
@@ -302,28 +301,29 @@ param_main(int argc, char *argv[])
 
 		if (!strcmp(argv[1], "reset")) {
 			if (argc >= 3) {
-				return do_reset((const char **) &argv[2], argc - 2);
+				return do_reset_specific((const char **) &argv[2], argc - 2);
 
 			} else {
-				return do_reset(nullptr, 0);
+				PX4_ERR("not enough arguments.");
```

Comment:
Useful for transition:
```suggestion
				PX4_ERR("not enough arguments (use 'param reset_all' to reset all).");
```

Reply:
Done with force-push